apps/challenges/aws_utils.py

```diff
@@ -697,3 +709,57 @@ def restart_workers_signal_callback(sender, instance, field_name, **kwargs):
                 instance.pk, field_name
             )
         )
+
+
+def create_eks_nodegroup(instance, cluster_name):
+    """
+    Creates a nodegroup when a EKS cluster is created by the EvalAI admin
```

Comment:
Can we please follow numpy docstring style as given here: https://numpydoc.readthedocs.io/en/latest/format.html ?

Reply:
Can we please add the docstrings in this format mentioned here:https://numpydoc.readthedocs.io/en/latest/format.html#sections
```
    """short description

    Arguments:
        x {type} -- description
        y {type} -- description

    Raises: (if applicable)
        error description and when it is raised

    Returns:
        z {type} -- description
    """
```